common/src/main/java/com/spredfast/kafka/connect/gcs/BytesRecordReader.java

```diff
@@ -148,6 +151,8 @@ private byte[] readBytes(int keySize, ReadContext context) throws IOException {
 			}
 			read += readNow;
 		}
+		String bytesAsString = Arrays.toString(bytes);
+		log.debug("debug readBytes bytesAsString {}", bytesAsString);
```

Comment:
Is the byte data likely to include personal or sensitive data? If so, this is potentially quite a dangerous log statement despite being 'DEBUG'.

Reply:
If we can be sure that this ill never go into production then this is fine.